endpoints-support.md

```diff
@@ -0,0 +1,581 @@
+|     **Migratable VC Domains**                                                                                                                    |
+|<sub>/rest/migratable-vc-domains</sub>                                                   | GET      | :heavy_minus_sign:   |                      |
+|<sub>/rest/migratable-vc-domains</sub>                                                   | POST     |    |                      |
+|<sub>/rest/migratable-vc-domains{id}</sub>                                               | PUT      |    |                      |
+|<sub>/rest/migratable-vc-domains{id}</sub>                                               | GET      |    |    |
+|<sub>/rest/migratable-vc-domains{id}</sub>                                               | DELETE   |    |    |
+|     **Network Set**                                                                                                                              |
+|<sub>/rest/network-sets</sub>                                                            |GET       | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/network-sets</sub>                                                            | POST     | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/network-sets/withoutEthernet</sub>                                            | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/network-sets/{id}</sub>                                                       | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/network-sets/{id}</sub>                                                       | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/network-sets/{id}</sub>                                                       | DELETE   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/network-sets/{id}/withoutEthernet</sub>                                       | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|     **OS Deployment Plans**                                                                                                                      |
+|<sub>/rest/os-deployment-plans/</sub>                                                    | GET      | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/os-deployment-plans/{id}</sub>                                                | GET      | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   |
+|     **OS Deployment Servers**                                                                                                                      |
+|<sub>/rest/os-deployment-servers/</sub>                                                  | GET      | :heavy_minus_sign:   |    |                      |
```

Comment:
Even that this endpoint is not implemented, but should the uri have the / at the end?

Reply:
Is marked as not implemented on the md file. Go to the branch and see the md file. Looking directly here in the review, it is not possible to see this.

Reply:
ok, the point of my comment was about the `/` at the end of URI, in documentation the URI not have the `/` at the end.